crates/dbsp/src/trace/mod.rs

```diff
@@ -240,6 +241,35 @@ where
     /// The removed tuples may not get deallocated instantly but they won't
     /// appear when iterating over the batch.
     fn truncate_keys_below(&mut self, lower_bound: &Self::Key);
+
+    /// Returns a uniform random sample of distincts keys from the batch.
```

Comment:
distinct.
You should emphasize that this does not consider weights. So it is mostly useful for sets.